attributes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dd-agent-builder": {
+      "omnibus-ruby_branch": "datadog-5.5.0"
+  },
+  "omnibus": {
+     "build_user_password": "DDBuildR0cks!"
```

Comment:
I tested out and I think we don't even need that, but I'm going to check with @degemer to be sure.